common/source/docs/common-uavcan-adapter-node.rst

```diff
@@ -60,8 +60,36 @@ An AP_Periph UAVCAN firmware supports these UAVCAN features:
 Setup
 ======
 
-Rangefinder
-------------
+AP_Periph f303-universal Firmware
+---------------------------------
+
+ The f303-Universal Firmware has the ability to be used for several devices but only one can be
+ enabled to use the single uart.
+ Once uploaded the default device is set to GPS, to use another device such as Rangefinder the
```

Comment:
Once uploaded the default device to be connected to the UART port is set to GPS....